diff -urN Guppi-0.40.3.org/src/libguppiuseful/Makefile.am Guppi-0.40.3/src/libguppiuseful/Makefile.am
--- Guppi-0.40.3.org/src/libguppiuseful/Makefile.am	Fri Jan 25 10:50:26 2002
+++ Guppi-0.40.3/src/libguppiuseful/Makefile.am	Fri Jan 25 10:59:47 2002
@@ -91,7 +91,7 @@
 	guppi-version.h 		\
 	guppi-xml.h
 
-noinst_HEADERS += \
+noinst_HEADERS = \
 	guppi-attribute-flavor-private.h
 
 #libguppiuseful_la_LDFLAGS = -version-info $(GUPPI_CURRENT):$(GUPPI_REVISION):$(GUPPI_AGE)
--- Guppi-0.40.3/configure.in~	Sun Sep 29 16:56:15 2002
+++ Guppi-0.40.3/configure.in	Sun Sep 29 16:56:15 2002
@@ -97,7 +97,6 @@
 AC_ISC_POSIX
 AC_STDC_HEADERS
 
-AM_ACLOCAL_INCLUDE(macros)
 AM_PROG_XML_I18N_TOOLS
 
 AM_MAINTAINER_MODE
--- Guppi-0.40.3/macros/Makefile.am~	Tue Jan  8 07:20:34 2002
+++ Guppi-0.40.3/macros/Makefile.am	Sun Sep 29 17:29:40 2002
@@ -30,13 +30,3 @@
 @MAINT@macros.dep: Makefile.am
 @MAINT@	@echo '$$(top_srcdir)/aclocal.m4: $(MACROS:%=macros/%)' > $@
 
-if INSIDE_GNOME_COMMON
-gnome_aclocaldir = $(datadir)/aclocal/gnome-macros
-
-gnome-macros.dep: Makefile.am
-	@echo '$$(top_srcdir)/aclocal.m4: $(MACROS:%=$(gnome_aclocaldir)/%)' > $@
-
-gnome_aclocal_DATA = $(MACROS) gnome-macros.dep gnome-common.m4 \
-	gnome-gettext.m4 autogen.sh
-
-endif
